File: FITCC.UI/admin_fitcc/lib/models/tim.dart
import 'package:admin_fitcc/models/takmicenje.dart';
import 'package:json_annotation/json_annotation.dart';

part 'tim.g.dart';

@JsonSerializable(explicitToJson: true)
class Tim {
    int timId;
    String? naziv;
    int? brojClanova;
    int? takmicenjeId;
    // Takmicenje takmicenje;

  Tim(
     this.timId,
     this.naziv,
     this.brojClanova,
     this.takmicenjeId,
    //  this.takmicenje,
  );

  factory Tim.fromJson(Map<String, dynamic> json) => _$TimFromJson(json);
  Map<String, dynamic> toJson() => _$TimToJson(this);
}

File: FITCCRS2App.Models/Models/Tim.cs
using System;
namespace FITCCRS2App.Models.Models
{
	public class Tim
	{
        public int TimId { get; set; }
        public string Naziv { get; set; } = string.Empty;
        public int? BrojClanova { get; set; }
        public int? TakmicenjeId { get; set; }
        public int? UserId { get; set; }
        public string Username { get; set; } = string.Empty;

        public virtual Takmicenje Takmicenje { get; set; } = new();
    }
}

